R/create_hexsticker.R

```diff
@@ -0,0 +1,31 @@
+#' @title Make hexagonal logo for package
+#'
+#' @description This function makes a hexsticker in INBO style for the provided
+#' package name.
+#'
+#' @param package_name package name that should be mentioned on the hexsticker
+#' @param filename filename to save the sticker (default saved in man/figures)
+#'
+#' @return A figure is saved in the working directory or provided path
+#'
+#' @export
+#'
+#' @family utils
+#'
+#' @examples
+#' create_hexsticker("checklist", filename = "hexsticker.png")
+
+create_hexsticker <-
+  function(package_name, filename = "man/figures/hexsticker.png") {
+  if (require("hexSticker") & require(showtext)) {
```

Comment:
you must use `requireNamespace()` instead of `require()`

Reply:
done